optuna/trial.py

```diff
@@ -156,14 +156,33 @@ def suggest_uniform(self, name, low, high):
 
         Example:
 
-            Suggest a dropout rate for neural network training.
+            Suggest a momentum for neural network training.
 
-            .. code::
+            .. testsetup::
 
-                >>> def objective(trial):
-                >>>     ...
-                >>>     dropout_rate = trial.suggest_uniform('dropout_rate', 0, 1.0)
-                >>>     ...
+                import numpy as np
+                from sklearn.model_selection import train_test_split
+
+                np.random.seed(seed=0)
+                X = np.random.randn(200).reshape(-1, 1)
+                y = np.random.randint(0, 2, 200)
+                X_train, X_test, y_train, y_test = train_test_split(X, y, random_state=0)
+
+            .. testcode::
+
+                import optuna
+                from sklearn.neural_network import MLPClassifier
+
+                def objective(trial):
+                    momentum = trial.suggest_uniform('momentum', 0.0, 1.0)
+                    clf = MLPClassifier(hidden_layer_sizes=(100, 50), momentum=momentum,
+                                        tol=1e-3, random_state=0)
```

Comment:
Please add `solver='sgd'` to arguments of `MLPClassifier()` because `momentum` is not used by the default solver (adam). c.f., https://scikit-learn.org/stable/modules/generated/sklearn.neural_network.MLPClassifier.html#sklearn.neural_network.MLPClassifier

In addition, can we remove the `tol` argument to simplify the example?

Reply:
Thanks! I fixed them.